src/windows/FileTransferProxy.js

```diff
@@ -116,9 +118,13 @@ exec(win, fail, 'FileTransfer', 'upload',
 
             // check if download isn't already cancelled
             var uploadOp = fileTransferOps[uploadId];
-            if (uploadOp && uploadOp.state === FileTransferOperation.CANCELLED) {
+            if (uploadOp && uploadOp.state === FileTransferOperation.CANCELLED || abortedOpIds.indexOf(uploadId) !== -1) {
```

Comment:
This condition is hard to read. Please encapsulate this logic in a descriptive function.